editor/src/core/model/performance-scripts.ts

```diff
@@ -33,3 +45,123 @@ export function useTriggerScrollPerformanceTest(): () => void {
   }, [dispatch])
   return trigger
 }
+
+export function useTriggerResizePerformanceTest(): () => void {
+  const dispatch = useEditorState(
+    (store) => store.dispatch as DebugDispatch,
+    'useTriggerResizePerformanceTest dispatch',
+  )
+  const metadata = useEditorState(
```

Comment:
since you are only using these in the callback, I think it's worth replacing this with a useRefEditorState, so the component using this hook doesn't rerender when the selected views or the metadata changes

Reply:
fixed